directories/src/redux/services/iota.js

```diff
@@ -1,19 +1,48 @@
 import IOTA from "iota.lib.js";
-import { IOTA_API_PROVIDER } from "../../config";
+import { IOTA_API_PROVIDER, IOTA_ADDRESS_LENGTH } from "../../config";
 import curl from "curl.lib.js";
+import _ from "lodash";
+import moment from "moment";
 
 const iota = new IOTA();
 
-const providerIota = new IOTA({
+const iotaProvider = new IOTA({
   provider: IOTA_API_PROVIDER
 });
 
 curl.init();
 const MAX_TIMESTAMP_VALUE = (Math.pow(3, 27) - 1) / 2;
 
+const checkIfClaimed = address =>
+  findTransactions([address]).then(transactions => {
+    const latestTransaction = transactions[0];
+    const attachedAt = latestTransaction.attachmentTimestamp;
+    const lastEpoch = moment()
+      .subtract(1, "year")
+      .valueOf();
+    return lastEpoch > attachedAt;
+  });
+
+const toAddress = string => string.substr(0, IOTA_ADDRESS_LENGTH);
+
+const findTransactions = addresses =>
```

Comment:
We probably should rename this to findTransactionObjects.  Normally I wouldn't care, but there's 2 iota API methods, one called "findTransactions" and the other called "findTransactionObjects" and they return different things.  It seems weird to be calling a method "findTransaction" that actually returns what "findTransactionObjects" returns.